pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -583,6 +584,18 @@ func readPostgresStmt(
 		} else {
 			schemaObjects.createTbl[schemaQualifiedName] = nil
 		}
+		for _, def := range stmt.Defs {
```

Comment:
Should we move this before when we store stmt in the map above? Let's also add a comment why we're doing this.

Can we use `NakedIntTypeFromDefaultIntSize` when resolving the int type?

Reply:
Done.